archive/rpm.ksy

```diff
@@ -1448,6 +1448,18 @@ enums:
     5098:
       id: arch_suffix
       -orig-id: RPMTAG_ARCHSUFFIX
+    5099:
+      id: spec
+      -orig-id: RPMTAG_SPEC
+    5100:
+      id: translation_url
+      -orig-id: RPMTAG_TRANSLATIONURL
+    5101:
+      id: upstream_releases
+      -orig-id: RPMTAG_UPSTREAMRELEASES
+    5102:
+      id: source_license
+      -orig-id: RPMTAG_SOURCELICENSE
```

Comment:
This tag is marked as *internal* (https://github.com/rpm-software-management/rpm/blob/9ed9d3fc/include/rpm/rpmtag.h#L383):

```c
    RPMTAG_SOURCELICENSE	= 5102, /* internal */
```

I used to declare such state in the tag identifier, according to the following description:

https://github.com/kaitai-io/kaitai_struct_formats/blob/c34d640bb8680b856ee2389f24f93e7dac8f1c28/archive/rpm.ksy#L458-L460

For example:

https://github.com/kaitai-io/kaitai_struct_formats/blob/c34d640bb8680b856ee2389f24f93e7dac8f1c28/archive/rpm.ksy#L545-L547

@armijnhemel Do you think it makes sense? If so, this would be:

```suggestion
    5102:
      id: source_license_internal
      -orig-id: RPMTAG_SOURCELICENSE
```

Reply:
Personally I would not use `_internal` (and just leave it out) but consistency is important. So feel free to replace!

Reply:
> Personally I would not use `_internal` (and just leave it out)

OK, so we may drop these later.